packages/text-input/src/types.ts

```diff
@@ -20,6 +20,10 @@ export interface TextInputProps {
   value?: any
   disabled?: boolean
   onChange?: React.ChangeEventHandler<HTMLInputElement>
+  onClick?: React.ChangeEventHandler<HTMLInputElement>
```

Comment:
Invalid event types. It should be `React.MouseEventHandler<HTMLInputElement>` or something like that